husky_components/label/main.js

```diff
@@ -29,6 +29,10 @@
  * @param {Function} [options.closeCallback] callback to execute if the close-button is clicked
  * @param {String} [options.insertMethod] insert method to use for inserting the label (append or prepend)
  * @param {String} [options.additionalLabelClasses] Additional classes which will be appended to the label
+ * @param {Array} [buttons] An array of buttons to add to the label
+ * @param {String} [buttons[].text] The text to show in the button
+ * @param {String} [buttons[].skin] The skin of the button e.g 'critical'
+ * @param {Function} [buttons[].callback] The callback to execute on click
```

Comment:
When trying this out I started with `title` and `callback` instead of `text` and `onClick`, can you doublecheck if these names are more common in the husky code base? And I just saw that the `callback` parameter mentioned in this line is not actually called `callback` 😄